app/pb_kits/playbook/pb_card/_card.jsx

```diff
@@ -36,13 +80,30 @@ const Card = ({
     [`highlight_${highlight.color}`]: highlight.color,
   })
 
+  // coerce to array
+  const leChildren = typeof(children) === 'object' && children.length ? children : [children]
+
+  const subComponentTags = (tagName) => {
+    return leChildren.filter((c) => {
+      return c.type && c.type.displayName === tagName
+    }).map((child, i) => {
+      return React.cloneElement(child, { key: `${tagName.toLowerCase()}-${i}` })
+    })
+  }
```

Comment:
Reuseable method that can be leverage later down the road as well. This method just needs the argument `tagName` (e.g. "Header") which correlates to the `Card.Header` subcomponent.